ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -66,6 +72,9 @@ export default class DashboardQuickActionsCard extends Component {
           buttonText: 'Issue leaf certificate',
           model: 'pki/role',
           route: 'vault.cluster.secrets.backend.pki.roles.role.generate',
+          nameKey: 'id',
```

Comment:
this should probably be `issuerName` (or `issuerRef` - which returns the name or ID depending on what's available since a name is optional)

Reply:
@hellobontempo why is this issuerName/issuerRefs? Should this be role name instead?

Reply:
The github diff had collapsed - so I just saw `issue certificate` and thought this was issuer related. 

yep this key can be removed because the default is already `name`

Reply:
No problem!